Herebyfile.mjs

```diff
@@ -696,7 +729,10 @@ export const runTestsAndWatch = task({
             if (!token.signaled) {
                 running = true;
                 try {
-                    await runConsoleTests(testRunner, "mocha-fivemat-progress-reporter", /*runInParallel*/ false, { token, watching: true });
+                    await runConsoleTests(testRunner, "mocha-fivemat-progress-reporter", /*runInParallel*/ false, {

```

Comment:
I would have expected it to be:
```ts
await runConsoleTests(
    testRunner,
    "mocha-fivemat-progress-reporter",
    /*runInParallel*/ false,
    { token, watching: true }
);
```

Reply:
Yeah, this just comes down to the algorithm that dprint uses; it sees the first parameter is on the same line so starts by keeping it there, then gets to the end and flows the object type over instead.